### ISSUE TITLE
`png.h` not actually used?

### ISSUE THREAD

**Issue by xylar** [ClimateGlobalChange/squadgen#6]
I spent quite a bit of time trying to understand why `libpng` was not actually being linked into `SQuadGen`.  I finally concluded that all the PNG functionality is provided by lodepng and the include here is not actually needed:
https://github.com/ClimateGlobalChange/squadgen/blob/master/src/SQuadGen.cpp#L17

@paullric, can you confirm if that's the case?  Is there an easy way to use an existing PNG library or is it best to just rely on lodepng?

**Comment by paullric** [comment]
Yes, I think the software no longer relies on libpng.  That should be removed from the list of requirements.
